specification/network/resource-manager/Microsoft.Network/stable/2022-11-01/virtualWan.json

```diff
@@ -8759,6 +8990,82 @@
       },
       "description": "ExpressRouteConnection list."
     },
+    "NetworkVirtualApplianceConnectionProperties": {
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "The name of the resource."
+        },
+        "provisioningState": {
+          "readOnly": true,
+          "$ref": "./network.json#/definitions/ProvisioningState",
+          "description": "The provisioning state of the NetworkVirtualApplianceConnection resource."
+        },
+        "asn": {
+          "type": "integer",
+          "readOnly": false,
+          "format": "int64",
+          "minimum": 0,
+          "maximum": 4294967295,
+          "description": "ASN."
+        },
+        "tunnelIdentifier": {
+          "type": "integer",
+          "readOnly": false,
+          "format": "int64",
+          "minimum": 0,
+          "maximum": 4294967295,
+          "description": "Unique identifier for the connection."
+        },
+        "bgpPeerAddress": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of bgpPeerAddresses for the NVA instances"
+        },
+        "enableInternetSecurity": {
```

Comment:
>enableInternetSecurity [](http://example.com/codeflow?start=9&length=22)

Consider using enum (with x-ms-enum) here instead of boolean, as enums are extensible without causing breaking changes in future. Foe example, here,

internetSecurityState as enum with values: "Enabled", "Disabled"